tests/integration/components/paper-sidenav-test.js

```diff
@@ -80,12 +81,12 @@ module('Integration | Component | paper sidenav', function(hooks) {
 
     await render(hbs`{{paper-sidenav open=isOpen lockedOpen=false}}`);
 
-    assert.ok(this.$('md-sidenav').hasClass('md-closed'));
+    assert.dom('md-sidenav').hasClass('md-closed');
 
     this.set('isOpen', true);
 
     return settled().then(() => {
```

Comment:
We can use 
```js
await settled();
assert.dom('md-sidenav').doesNotHaveClass('md-closed');
```

instead of the promise then.

Can you convert that, please?

Thank for all your efforts on removing jquery. It's really valuable!